packages/workflow/package.json

```diff
@@ -28,38 +28,38 @@
     "@availity/mock-server": "^7.0.0",
     "@availity/workflow-logger": "^5.5.0",
     "@availity/workflow-upgrade": "^5.4.2",
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.7.2",
     "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/runtime": "^7.0.0",
-    "@testing-library/jest-dom": "^4.0.0",
+    "@babel/runtime": "^7.7.2",
+    "@testing-library/jest-dom": "^4.2.3",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^24.0.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-jsx-remove-data-test-id": "^2.0.0",
     "babel-plugin-root-import": "^6.4.1",
     "babel-preset-react-app": "^9.0.2",
     "case-sensitive-paths-webpack-plugin": "^2.1.2",
-    "chalk": "^2.3.2",
-    "copy-webpack-plugin": "^5.0.4",
-    "cross-spawn": "^6.0.5",
-    "css-loader": "2.1.1",
+    "chalk": "^3.0.0",
+    "copy-webpack-plugin": "^5.0.5",
+    "cross-spawn": "^7.0.1",
+    "css-loader": "3.2.0",
     "debug": "^4.0.1",
     "del": "^5.1.0",
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
     "envinfo": "^7.4.0",
     "escape-string-regexp": "^2.0.0",
-    "eslint": "^5.16.0",
-    "eslint-config-availity": "^5.2.1",
-    "eslint-loader": "^2.1.2",
-    "execa": "^2.0.3",
-    "figures": "^3.0.0",
+    "eslint": "^6.6.0",
```

Comment:
Are we okay to upgrade to v6 without everything breaking? Just want to make sure this is noted so you can test all the scripts.